share/ramble/qa/run-unit-tests

```diff
@@ -42,4 +42,4 @@ bin/ramble help -a
 #-----------------------------------------------------------
 # Run unit tests with code coverage
 #-----------------------------------------------------------
-$coverage_run $(which ramble) unit-test -x --verbose
+$coverage_run $(which ramble) unit-test -x --verbose -m "not long"
```

Comment:
Do you have strong thoughts about which way round we want the unit test behavior by default? Should it do `long` or `not long` by default for a developer testing on their local machine?